styles.css

```diff
@@ -0,0 +1,112 @@
+@media only screen and (min-width: 25rem) {
+  * {
```

Comment:
- [ ] Please make sure your project is mobile-first and your design matches the Figma design :recycle: 

  - Your design:
![yourD](https://user-images.githubusercontent.com/97834160/211959041-3ab8340e-dace-433f-8127-b440a71b2e80.png)


  - Figma design:

![FigmaD](https://user-images.githubusercontent.com/97834160/211959068-53aecb2d-7460-4922-81a4-65e88091a1df.png)

Reply:
Hi @GedeonTS ,

I appreciate the review and I'm currently working on the changes 🙏 

Just have a quick question. The screenshot attached to your comment on this thread shows my project without any style (as if there were no styles.css file). But as you can see in the screenshot I'm attaching to this comment, when I open my project it shows the proper styling 😟 

I'm a bit confused about what to do in this situation. I'd appreciate it if you could give me some insight on the matter 😸 

![image](https://user-images.githubusercontent.com/107666488/211962014-1f6bf9b2-0e52-42ae-a71d-ddc7e3bfa867.png)